newsfragments/43183.significant.rst

```diff
@@ -0,0 +1,5 @@
+Remove TaskContextLogger
+
+We introduced this as a way to inject messages into task logs from places
+other than the task execution context.  We later realized that we were better off
+just using the Log table.
```

Comment:
Am I correct in thinking we wanted to use this feature to show triggered logs and scheduler logs into Task logs?

Do we have a replacement? I think Triggerer logs now appear in Webserver. Worth mentioning it if that's right.

Reply:
this is mostly unrelated to triggerer logging. scheduler was the only place it was used. but btw it wasn't ever actually a public feature. the class itself was private (so that it could be changed easily and it was never intended to be used by users.) but there was an attr that existed on log handler that was technically public.